src/printer.js

```diff
@@ -2411,6 +2402,30 @@ function printFlowDeclaration(path, parts) {
   return concat(parts);
 }
 
+function getFlowVariance(path, options) {
+  if (!path.variance) {
+    return null;
+  }
+
+  // Babylon 7.0 currently uses variance node type, and flow should
+  // follow suit soon:
+  // https://github.com/babel/babel/issues/4722
+  const variance = (options.parser === "babylon")
+    ? path.variance.kind
+    : path.variance;
```

Comment:
Since flow is going to follow soon, can you change this to be

```js
const variance = path.variance.kind || path.variance;
```

this way when we update flow it's going to work as is instead of breaking this test.